CHANGELOG.md

```diff
@@ -49,10 +49,10 @@ Versioning](https://semver.org/spec/v2.0.0.html).
       in version `0.21.0-alpha.2`.
 - Rename `chwaddr` to `chaddr` because the field names within the `Bootp` structure
   of the giganto-client have changed.
-- Changed cluster realted configuration field names.
+- Changed cluster related configuration field names.
   - `peer_address` to `addr_to_peers`
   - `address` in `peers` to `addr` and `host_name` in `peers` to `hostname`
-- Removed `unsafe` block in `write_run_tcpdump` while creating a temorary file.
+- Removed `unsafe` block in `write_run_tcpdump` while creating a temporary file.
 - Remove migration code less than `0.15.3`
```

Comment:
I think these should be in the section "Removed". What do you think about that?

Reply:
done